convert2rhel/pkghandler.py

```diff
@@ -273,29 +270,25 @@ def get_third_party_pkgs():
     Get all the third party packages (non-Red Hat and non-original OS-signed)
     that are going to be kept untouched.
     """
-    third_party_pkgs = get_installed_pkgs_w_different_fingerprint(
-        system_info.fingerprints_orig_os + system_info.fingerprints_rhel
-    )
+    third_party_pkgs = get_installed_pkgs_w_different_key_id(system_info.key_ids_orig_os + system_info.key_ids_rhel)
 
     return third_party_pkgs
 
 
-def get_installed_pkgs_w_different_fingerprint(fingerprints, name="*"):
+def get_installed_pkgs_w_different_key_id(key_ids, name="*"):
     """Return list of all the packages (yum.rpmsack.RPMInstalledPackage objects in case
     of yum and hawkey.Package objects in case of dnf) that are not signed
-    by the specific OS GPG keys. Fingerprints of the GPG keys are passed as a
-    list in the fingerprints parameter. The packages can be optionally
+    by the specific OS GPG keys. key_ids of the GPG keys are passed as a
+    list in the key_ids parameter. The packages can be optionally
```

Comment:
Stuff like this exists throughout
```suggestion
    by the specific OS GPG keys. GPG key IDs are passed as a
    list in the key_ids parameter. The packages can be optionally
```

Reply:
```suggestion
    by the specific OS GPG keys. GPG Key IDs of the GPG keys are passed as a
    list in the key_ids parameter. The packages can be optionally
```